tests/core/test_reader.py

```diff
@@ -22,9 +22,12 @@
 skip_windows = pytest.mark.skipif(os.name == 'nt', reason='Test fails on Windows')
 
 
-def test_get_reader_fail():
+def test_get_reader_fail(tmp_path):
     with pytest.raises(ValueError):  # noqa: PT011
         pv.get_reader("not_a_supported_file.no_data")
+    msg = "`pyvista.get_reader` does not support reading from directory:\n\t"
+    with pytest.raises(ValueError, match=msg):
+        pv.get_reader(str(tmp_path))
```

Comment:
nit: It is better to match the name of the argument to be assigned with the name of the variable.
```suggestion
    match= "`pyvista.get_reader` does not support reading from directory:\n\t"
    with pytest.raises(ValueError, match=match):
        pv.get_reader(str(tmp_path))
```